dev/postgres-docker/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3.5"
+services:
+  postgres:
+    build:
+      context: .
+      dockerfile: postgresql.dockerfile
```

Comment:
Why not using [hzibraunschweig/sormas-postgres](https://hub.docker.com/r/hzibraunschweig/sormas-postgres) ?

Reply:
Good point. I did not know that the images from SORMAS-Docker are also published at docker hubs, which was the reason why I included the recipe for the PostgreSQL container variant such that one does not need to clone the SORMAS-Docker repo as well. The latest commit I pushed will use the mentioned existing image in the `docker-compose.yml`.